packages/twenty-front/src/modules/ui/input/components/TextInput.tsx

```diff
@@ -1,141 +1,26 @@
-import {
```

Comment:
Shouldn't we keep just TextInputV2 ?

Reply:
You mean get rid of TextInput altogether now? 
I want to take more time before replacing them where the hotkey scopes are not disabled to make sure I am dealing with them correctly

Reply:
ok